src/symbol/sprite_atlas.js

```diff
@@ -14,9 +16,11 @@ class AtlasImage {
     }
 }
 
-class SpriteAtlas {
+class SpriteAtlas extends Evented {
```

Comment:
This isn't a comment about the changes in this PR, but just noting while I'm here: having never worked in this code before, I found it confusing, at first, to make sense of how/why `SpriteAtlas` works. E.g., "what's with the `this.copy()`?", "what's a `rect`?", "why does `AtlasImage` not have a reference to the underlying data?"

Reading more widely through the codebase, I think I've satisfied my questions, but this might be a nice time to add a brief, high-level description of what `SpriteAtlas` is/does.  OTOH ticketing that separately is also totally fair 😄